### ISSUE TITLE
Test deployment from the 1.5.x branch

### ISSUE THREAD

**Issue by jwhitlock** [mozilla/ichnaea#837]
The 1.5.x branch was the official branch. To get things working, we've deployed from ckolos/dammit. Those changes have been merged back to the 1.5.x branch, and the remaining task is to prove that we can deploy from there.

**Comment by jwhitlock** [comment]
Assigning to @ckolos. I don't think this is urgent, but should be done before further changes to the 1.5.x series.

**Comment by ckolos** [comment]
this has been deployed to stage. Pending review and approval, we can push this to prod.

**Comment by jwhitlock** [comment]
https://location.stage.mozaws.net/__version__ shows:

```
{"commit": "HEAD", "tag": "master", "version": "1.5.1", "source": "https://github.com/mozilla/ichnaea"}
```

while [production](https://location.services.mozilla.com/__version__) shows:

```
{"commit": "a1e1570f12e25998c028f166ec0329dab6b15425", "tag": "20170606213300", "version": "1.5.1", "source": "https://github.com/mozilla/ichnaea"}
```

I appreciate the commit hash being preserved when a tag is used, and I'd like to know what makes for a good prod push from this branch before we need it. After some discussion, I tagged ``20190813103806``, and I'd like to get that deployed to stage.

**Comment by jwhitlock** [comment]
@ckolos deployed the tag to stage, and the commit is still ``HEAD``. There may be some caching, since the commit is identical. We'll try deploying from a tag first next time, and see if we get different results.

The changes from the last deployment are CircleCI configuration, the test location database, and some other non-functional changes. I think we've learned what we can, and a prod deploy is not necessary at this time.

**Comment by jwhitlock** [comment]
Now that I look into it, it is a bit stranger. Prod says it is on commit a1e1570f12e25998c028f166ec0329dab6b15425 of tag `20170606213300`, but if you look at the changes since then (https://github.com/mozilla/ichnaea/compare/20170606213300...20190813103806), it includes stuff that is definitely deployed to prod, like the updated API key paragraph.

I think the issue is not caching, but that the version file is not being generated correctly. I think this is a "won't fix" for the 1.5.x branch, but that we should get it working in the new 2.x deployment system.
